.github/workflows/deploy-dev-gcp.yaml

```diff
@@ -0,0 +1,38 @@
+name: Deploy to dev
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - dev
+env:
+    CI: true
+    TZ: Europe/Amsterdam
+
+jobs:
+    deploy-dev:
+        name: Deploy application to dev
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+            - name: Setup node
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 16
+                  cache: npm
+            - name: Install dependencies
+              run: npm ci
+            - name: Run tests
+              run: npm run test
+            - name: Build application
+              run: npm run build:gcp
```

Comment:
We don't have a `build:gcp` command, only `build`

Reply:
Fixed